src/clj/rems/schema_base.clj

```diff
@@ -35,3 +54,27 @@
    :event/time DateTime
    :event/actor UserId
    :application/id s/Int})
+
+(s/defschema UserWithAttributes
+  {:userid UserId
+   :name (s/maybe s/Str)
+   :email (s/maybe s/Str)
+   (s/optional-key :organizations) [OrganizationId]
+   (s/optional-key :notification-email) (s/maybe s/Str)
+   (s/optional-key :researcher-status-by) s/Str
+   s/Keyword s/Any})
+
+(s/defschema OrganizationOverview
+  (merge OrganizationId
+         {:organization/short-name LocalizedString
+          :organization/name LocalizedString}))
+
+(s/defschema OrganizationFull
+  (merge OrganizationOverview
+         {(s/optional-key :organization/modifier) UserWithAttributes
+          (s/optional-key :organization/last-modified) DateTime
+          (s/optional-key :organization/owners) [UserWithAttributes]
+          (s/optional-key :organization/review-emails) [{:name LocalizedString
+                                                         :email s/Str}]
+          (s/optional-key :enabled) s/Bool
+          (s/optional-key :archived) s/Bool}))
```

Comment:
Should this technically be part of organization API or here?

Reply:
OrganizationFull and OrganizationOverview are used in rems.db.organizations, which is why I moved them here. I guess they could also be in the db ns?

Reply:
Since Organization is the latest, it has pretty much identical DB model as API model. I guess this is fine too though I would expect that the `schema-base` would remain "slim" while the DB and API are the "fullest".

Reply:
Yeah I'd prefer a slim `schema-base` too. I'm not that pleased with how these ended up here, but it's better than `rems.api.schema`. It makes the API sharing visible, at least.